lib/cli/src/logging.rs

```diff
@@ -1,14 +1,44 @@
 //! Logging functions for the debug feature.
-use crate::utils::wasmer_should_print_color;
-use anyhow::Result;
-use fern::colors::{Color, ColoredLevelConfig};
-use std::time;
 
-/// The debug level
-pub type DebugLevel = log::LevelFilter;
+/// Subroutine to instantiate the loggers
+#[cfg(feature = "tracing")]
```

Comment:
I think it's fine to just switch to tracing full stop and remove the old log system.

Reply:
the spans all have `skip-all` turned on by default - which means the number of fields have to be explicitly added and won't grow, but it might be an idea to add a feature flag for super noisy syscalls.

to be honest i think we should do the opposite though as I commended out the `thread_local_get` syscall debug and trace messages because they were so noisy... but then later (as in a few days ago) when I put the spans on i noticed there were millions of them now because of `tokio` going nuts with locals.

i think we should leave it noisy when in tracing mode, make it "useful" in debug level.... and optimize the code so that less syscalls are made. i mean the syscalls will be killing performance anyway